tests/pubsub/utils.nim

```diff
@@ -512,6 +516,21 @@ proc addIDontWantObservers*[T: PubSub](
 
   return allMessages
 
+proc findAndStopPeers*[T: PubSub](
+    nodes: seq[T], peers: seq[PeerId], topic: string, handler: TopicHandler
+) {.async.} =
+  for i in 0 ..< nodes.len:
+    let node = nodes[i]
+    if peers.anyIt(it == node.peerInfo.peerId):
+      node.unsubscribe(topic, voidTopicHandler)
+      await node.stop()
```

Comment:
i am not really sure about this func, maybe it should only do 1 thing (unsubscribe). 
if node is stopped then it will also trigger mesh to rebalanced?